pkg/doc/signature/proof/jws.go

```diff
@@ -101,7 +101,9 @@ func prepareJWSProof(suite signatureSuite, proofOptions map[string]interface{})
 
 	delete(proofOptionsCopy, jsonldJWS)
 	delete(proofOptionsCopy, jsonldProofValue)
-	delete(proofOptionsCopy, jsonldType)
+	// TODO not any where mentioned in RFCs to exclude type, this is causing issue while validating signature from
+	// other systems []
+	// delete(proofOptionsCopy, jsonldType)
```

Comment:
should add a flag to execute `delete()` here instead of commenting out this line

Reply:
Refer: https://github.com/digitalbazaar/jsonld-signatures/blob/master/lib/suites/LinkedDataSignature.js#L194-L196

We are deleting it from copy to calculate hash, flag will change the hash. Refer above code from JSON-LD specifications.

Reply:
@sudeshrshetty they must have changed spec then since I remember clearly that you needed to remove proof section before calculating JSON-LD signature

Reply:
I see, they are not excluding "proof" section they are excluding parts of proof 
delete proof.jws;
delete proof.signatureValue;
delete proof.proofValue;

Reply:
@sandrask @sudeshrshetty yes, they recently changed this in the code (`LinkedDataSignature.js`) and in the spec as well (https://w3c-ccg.github.io/ld-proofs/).